nomad/structs/csi_volumes.go

```diff
@@ -0,0 +1,341 @@
+package structs
```

Comment:
Could you merge this file into nomad/structs/csi.go?

Reply:
done